Assets/Project/Modules/PlayerAnchor/Scripts/Anchor/VFXAnchorView.cs

```diff
@@ -151,8 +151,9 @@ public void PlayKickedAnimation(float duration)
 
         public void PlayCarriedAnimation()
         {
+            _specialMotionsTransform.DOLocalRotateQuaternion(Quaternion.identity, 0.2f);
             //_carryTrail = _particleFactory.Create(_carryTrailParticleType, Vector3.zero, Quaternion.identity, _vfxParent)
-                //.gameObject.GetComponent<InterpolatorRecycleParticle>();
+            //.gameObject.GetComponent<InterpolatorRecycleParticle>();
```

Comment:
Might have missed these comments

Reply:
These are yours xd, I just needed to open the script to add a line of code. Might have unindented something without realizing